adapters/timm/classification/efficientnet/dataloop.json

```diff
@@ -0,0 +1,1194 @@
+{
+  "name": "efficientnet",
+  "displayName": "EfficientNet",
+  "version": "0.1.17",
+  "scope": "project",
+  "description": "EfficientNet application for image classification.",
+  "attributes": {
+    "Provider": "Dataloop",
+    "Category": "Model",
+    "Computer Vision": "Classification",
+    "Media Type": [
+      "Image"],
+    "Libraries": "Torch",
+    "Deployed By": "Dataloop"
+  },
+  "codebase": {
+    "type": "git",
+    "gitUrl": "https://github.com/dataloop-ai-apps/torch-models.git",
+    "gitTag": "0.1.16"
+  },
+  "components": {
+    "computeConfigs": [
+      {
+        "name": "efficientnet-deploy",
+        "versions": {
+          "dtlpy": "1.90.39"
+        },
```

Comment:
i think you can remove 
        "versions": {
          "dtlpy": "1.90.39"
        },